tests/vmcheck/overlay.sh

```diff
@@ -45,6 +45,13 @@ rsync -rlv $INSTTREE/usr/ vmcheck/usr/
 if [ -d $INSTTREE/etc ]; then # on CentOS, the dbus service file is in /usr
   rsync -rlv $INSTTREE/etc/ vmcheck/usr/etc/
 fi
+
+# ✀✀✀ BEGIN hack to get --selinux-policy (https://github.com/ostreedev/ostree/pull/1114) ✀✀✀
```

Comment:
Maybe only do this if we don't detect `--selinux-policy` in the target, i.e. `if ! ostree commit --help | grep -q -e --selinux-policy`?